torch/distributed/_tensor/ops/pointwise_ops.py

```diff
@@ -408,6 +408,20 @@
 def pointwise_strategy(
     mesh: DeviceMesh, op_schema: OpSchema, linearity: bool = False
 ) -> OpStrategy:
+    """
+    Pointwise operators can follow the strategy of its first argument
+    For example, c = add(a, b). If a is sharded, we can follow the
+    sharding of a to generate the strategy for b.
+
+    Args:
+        mesh (DeviceMesh): device mesh for pointwise ops
+        op_schema (OpSchema): schema of the operator to generate strategy for
+        linearity (bool): whether this op supports broadcasting or not
```

Comment:
`linearity` is actually not about broadcasting, it's about a addition semantics:

i.e. if the operator itself could support `op(a) + op(b) = op (a + b)`